Small updates regarding position information (#33)
nixpkgs-architecture/simple-package-paths: README.md

# Interactions
[interactions]: #interactions

> - `nix edit` and search.nixos.org is unaffected, since they rely on `meta.position` to get the file to edit, which still works

Reviewer: ```suggestion
- `nix edit` and search.nixos.org are unaffected, since they rely on `meta.position` to get the file to edit, which still works
```

    - Many packages fit multiple categories, leading to multiple locations to search through instead of one
  - There's other better ways of discovering similar packages, e.g. [Repology](https://repology.org/)
- This breaks `builtins.unsafeGetAttrPos "hello" pkgs`. Counter-arguments:

>   - This functionality is impure and therefore the result can't be assumed to stay the same over time

Reviewer: ```suggestion
  - This functionality is accepted to be unreliable and therefore it does not seem reasonable to assume that it stays the same over time either.
```

Purity is meaningless without a defining equality, and we really don't need to get into that here.
I'd like to avoid a situation where we just call everything we don't like impure. That's not accurate and devalues the meaning of the word.

>   - This functionality is impure and therefore the result can't be assumed to stay the same over time

Reviewer: ```suggestion
```

Even then this is actually a non-sequitur.

>   - This functionality is impure and therefore the result can't be assumed to stay the same over time

Reviewer: Though we can say this:

```suggestion
  - We have to draw a line as to what constitutes the public interface of Nixpkgs. We have decided that making attribute position information part of that is not productive. For context, this information is already accepted to be unreliable at the language level, noting the `unsafe` part of the name.
```

Author: This sounds good to me too
